README.rst

```diff
@@ -173,13 +173,20 @@ Developing Renku
 ================
 
 For development it's convenient to install ``renku`` in editable mode. This is
-still possible with ``pipx``. First clone the repository and then do:
+still possible with ``pipx``. First clone the repository and run the setup:
+
+::
+
+    $ python setup.py bdist_wheel
```

Comment:
I'm not sure this is needed? if you install in editable mode the wheel is ignored anyway

Reply:
Yeah I wasn't sure about this. I kept getting an error until I manually run the setup but it looks strange we have to do it manually :thinking: